### ISSUE TITLE
dev branch?

### ISSUE THREAD

**Issue by Consolatis** [richrd/suplemon#214]
Readme states PRs should be based on the dev branch so I based my feature branches on top of that.
Now I wanted to send the first smaller ones but the dev branch is gone, resulting in me being confused and scratching my head.

Accident? Deleted on purpose in favor of rewrite-dev?
I am fine with either rebasing or even completely rewriting the features but need some advice on how to proceed.

**Comment by richrd** [comment]
Oops, I deleted it after the last release and forgot to recreate it. I went ahead and created it based on the master branch.

**Comment by Consolatis** [comment]
Thanks for recreating the branch :)

What are your plans for the rewrite branches, should I develop against them instead of the current dev branch?

**Comment by richrd** [comment]
I'd definitely recommend the current dev and master. The rewrite stuff is extremely primitive and doesn't even really run yet. It does have a basic widget and layout engine and a curses backend implementation. I should get back to them but I'm in a bit of a limbo with what to do with the architecture. Need to plan something out before continuing.

**Comment by Consolatis** [comment]
Alright, dev it is then.
If I am done with the stuff I currently have in my queue I might have a look at your rewrite branch.

JFYI: I really appreciate the rationale of the project:
> For them (like for me) nano was the weapon of choice. But nano feels clunky and it has its limitations. 
I have been in the same boat.
Using Vim with its modes is not intuitive at all. For me, Emacs was better because you can just use it. But once you start using it more and more most of the time you have to think about what kind of crazy key bindings you have to use to get something specific done.
So for me as well, nano was the weapon of choice, despite its flaws.
Suplemon indeed feels like nano but allows me to extend and customize it the way I want.
So to sum it up: just leaving a honest "thank you" here :)
